hack/docgen.go

```diff
@@ -1,3 +1,5 @@
+// +build !fields
```

Comment:
This file imports `"github.com/argoproj/argo/cmd/argo/commands"`, so it will in effect build the entire codebase into the binary. This takes a lot of time and it will require generating static files, which takes even longer. Since this is unnecessary for the script in this PR, don't include this file when we're building for this purpose.

Reply:
I'm fine with this. Can I ask we just fix the docs?